andela/ah-the-jedi-frontend
# #165305214 Setup Empty React Project

## What does this PR do?

    Creates an empty react app

## Description of the tasks to be completed?

    create a react project from scratch using Webpack 4, Babel, and the Airbnb linting option.

## How should this be manually tested?

    Clone repo at https://github.com/andela/ah-the-jedi-frontend.git, 
    Check out to ch-setup-project-165305214 and run in your terminal " npm start "

What are the relevant pivot tracker stores?

[#165305214](https://www.pivotaltracker.com/story/show/165305214)
